From 57a409441bef013f636ac8e72836c19ecb27c1b9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=8D=C3=B1igo=20Huguet?= <ihuguet@riseup.net>
Date: Fri, 12 Dec 2025 20:15:51 +0100
Subject: [PATCH] release: bump version to 1.52.2

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 847f72b5..4b6d9453 100644
--- a/meson.build
+++ b/meson.build
@@ -5,7 +5,7 @@ project(
 # NOTE: When incrementing version also add corresponding
 #       NM_VERSION_x_y_z macros in
 #       "src/libnm-core-public/nm-version-macros.h.in"
-  version: '1.52.1',
+  version: '1.52.2',
   license: 'GPL2+',
   default_options: [
     'buildtype=debugoptimized',
-- 
2.34.1

